.github/workflows/ci-test-xtgeo-cibuildwheel.yml

```diff
@@ -0,0 +1,67 @@
+# Test building some cibuildwheel on pull requests (simplified testing)
+
+name: cibuildwheel
+
+on:
+  pull_request:
+    branches: [master]
+
+env:
+  CIBW_BEFORE_ALL_LINUX: "sh scripts/swig_manylinux.sh"
+  CIBW_BEFORE_ALL_WINDOWS: "choco install -y --no-progress --allow-empty-checksums -r swig --version=4.0.1"
+  CIBW_BEFORE_ALL_MACOS: brew install swig
+
+  CIBW_BEFORE_TEST: |
+    pushd {project}
+    pip install -r requirements/requirements.txt
+    pip install -r requirements/requirements_test.txt
+
+  # 3.9 have trouble with install pytables on Mac and Win, due to missing wheel
+  # so full testing on those platforms are postponed.
+  CIBW_TEST_COMMAND_MACOS: |
+    pushd {project}
+    rm -rf ../xtgeo-testdata
+    git clone --depth 1 https://github.com/equinor/xtgeo-testdata ../xtgeo-testdata
+    pytest tests/test_surface --disable-warnings
```

Comment:
So, are we simply not running any tests that use `hdf` format, and hence `xtgeo` will work for anyone unless they import/export `hdf`?

Reply:
Rewrote actions, and only skipped well tests for MacOS and Windows. Since wells to HDF is currently very experimental (it is pandas that use tables in lazyload mode) no users will in practice experience any problems with missing tables for the current version I think; hence deplying 3.9 for Windows and Macos is defendable. But of course could consider skipping Python 3.9 for Windows and Mac PYPI deploy of xtgeo until Tables wheels exists at PYPI. What do you think?

Reply:
>Rewrote actions, and only skipped well tests for MacOS and Windows.

Better!

I'm not sure if I have that strong opinions. I would think waiting would be the most user friendly, but I suppose for advanced users, they could compile tables themselves if they _really_ want it for python 3.9 (if they're that advanced, they could probably simply compile xtgeo as well though). If this is low impact then I don't mind.

Reply:
OK, I will keep as is